packages/turf-invariant/test.js

```diff
@@ -208,3 +208,17 @@ test('invariant#getGeomType', t => {
     t.throws(() => invariant.getGeomType(collection, 'featureCollection not valid'));
     t.end();
 });
+
+// https://github.com/Turfjs/turf/issues/853
+test('null geometries', t => {
+    const nullFeature = {
+        type: 'Feature',
+        properties: {},
+        geometry: null
+    }
+    t.equal(invariant.getGeom(null), null);
```

Comment:
@DenisCarriere why supporting `null` values? Reading the [specs](https://tools.ietf.org/html/rfc7946#section-3) it seems the GeoJSON objects _is_ an object and _has_ to have a `type` member (which in my understanding is the only requirement to be a valid GeoJSON object)